grakn-kb/src/main/java/ai/grakn/kb/internal/computer/GraknComputerImpl.java

```diff
@@ -88,8 +88,10 @@ public ComputerResult compute(@Nullable VertexProgram program, @Nullable MapRedu
             if (mapReduce != null) graphComputer.mapReduce(mapReduce);
             applyFilters(types, includesRolePlayerEdges);
             return graphComputer.submit().get();
-        } catch (InterruptedException | ExecutionException e) {
```

Comment:
`GraknComputerImpl` now handles  `ExecutionException` in a cleaner manner. Before, the method will throw a `RuntimeException` which will wrap `ExecutionException` which wrap the actual `Exception`. Now, the `RuntimeException` will wrap the actual `Exception` directly without any intermediary `ExecutionException`